pages/README.md

```diff
@@ -0,0 +1,50 @@
+# React + TypeScript + Vite
```

Comment:
I think we can upgrade readme by explaining what theses pages are for, how to test changes etc etc